java/src/jmri/jmrit/display/layoutEditor/LayoutEditorTools.java

```diff
@@ -808,18 +821,18 @@ private void setLogicThroatContinuing() {
         Sensor occupancy = block.getOccupancySensor();
         if (occupancy == null) {
             JOptionPane.showMessageDialog(setSignalsFrame,
-                    java.text.MessageFormat.format(rb.getString("InfoMessage4"),
+                    MessageFormat.format(rb.getString("InfoMessage4"),
```

Comment:
Since rb refers to the same ResourceBundle as Bundle in this package uses, all uses of ```MessageFormat.format(rb.getString("SOMETHING"), TOKENS...)``` can be replaced with ```Bundle.getMessage("SOMETHING", TOKENS...)```

Reply:
Is this true everywhere in LayoutEditor sources (package)? If so I'll find/fix this everywhere there.

Reply:
I was able to remove it everywhere except LayoutEditorXml… for some reason its complaining about unable to convert string to local… I'll leave it as-is for now with a TODO for later.

Reply:
The way we designed Bundle, it is expected that jmri.jmrit.display.layoutEditor.configureXml.LayoutEditorXml would not be able to use jmri.jmrit.display.layourEditor.Bundle. You would need to copy Bundle into the configureXml package and make the copy a subclass of Bundle in the parent directory.